File: app/src/main/java/com/treelzebub/umap/api/discogs/DiscogsConstants.java
package com.treelzebub.umap.api.discogs;

/**
 * Created by Tre Murillo on 3/29/15
 */
public class DiscogsConstants {

    public static final String BASE_URL = "https://api.discogs.com";
    public static final String REQUEST_TOKEN_URL = BASE_URL + "/oauth/request_token";
    public static final String ACCESS_TOKEN_URL = BASE_URL + "/oauth/access_token";
    public static final String AUTHORIZATION_URL = "https://www.discogs.com/oauth/authorize";
    public static final String AUTH_URL_APPEND = "?oauth_token=";
    public static final String CALLBACK_URL = "oauth://umap.treelzebub.net";

    public static final String CONSUMER_KEY = "JfdZojxprrQHOqIpHwpV";
    public static final String CONSUMER_SECRET = "uigJkstmlBPlLBFMoxAApEHgIdrCGQsx";
    public static final String SIGNATURE_METHOD = "PLAINTEXT";

    public static final String IDENTITY_URL = "/oauth/identity";
    public static final String PROFILE_URL = "/users"; // + /{username}

    private DiscogsConstants() {
    }

}

File: app/src/main/kotlin/com/treelzebub/umap/api/discogs/constants/DiscogsConstants.kt
package com.treelzebub.umap.api.discogs.constants

/**
 * Created by Tre Murillo on 5/28/15
 * Copyright(c) 2015 Level, Inc.
 */

public val BASE_URL: String = "https://api.discogs.com"
public val REQUEST_TOKEN_URL: String = BASE_URL + "/oauth/request_token"
public val ACCESS_TOKEN_URL: String = BASE_URL + "/oauth/access_token"
public val AUTHORIZATION_URL: String = "https://www.discogs.com/oauth/authorize"
public val AUTH_URL_APPEND: String = "?oauth_token="
public val CALLBACK_URL: String = "oauth://umap.treelzebub.net"

public val CONSUMER_KEY: String = "JfdZojxprrQHOqIpHwpV"
public val CONSUMER_SECRET: String = "uigJkstmlBPlLBFMoxAApEHgIdrCGQsx"
public val SIGNATURE_METHOD: String = "PLAINTEXT"

public val IDENTITY_URL: String = "/oauth/identity"
public val PROFILE_URL: String = "/users" // + /{username}